includes/admin/class-education.php

```diff
@@ -191,4 +201,27 @@ public static function is_installed( $requires ) {
 
 		return false;
 	}
+
+	/**
+	 * Checks if the user has access to a module
+	 *
+	 * @param string $module
+	 * 
+	 * @return bool
+	 */
+	private static function has_access_to( $module ) {
+		$licenses = get_option( 'wpbdp_licenses', array() );
+
+		if ( ! is_array( $licenses ) || ! isset( $licenses[ 'module-business-directory-' . $module ] ) ) {
+			return false;
+		}
+
+		$license = $licenses[ 'module-business-directory-' . $module ];
+
+		if ( ! is_array( $license ) || ! isset( $license['status'] ) || $license['status'] !== 'valid' ) {
+			return false;
+		}
+
+		return true;
```

Comment:
Hi @sorinmarta,

Since the condition `!is_array( $license ) || !isset( $license['status'] ) || $license['status'] !== 'valid'` will naturally return a boolean value, we can simplify the code by removing `false` and `true` and directly returning the result like this:

```PHP
/**
 * Checks if the user has access to a module
 *
 * @param string $module
 * 
 * @return bool
 */
private static function has_access_to( $module ) {
    $licenses = get_option( 'wpbdp_licenses', array() );
    $license  = isset( $licenses[ 'module-business-directory-' . $module ] ) ? $licenses[ 'module-business-directory-' . $module ] : null;

    return is_array( $license ) && isset( $license['status'] ) && $license['status'] === 'valid';
}
```